packages/sitecore-jss/src/i18n/graphql-dictionary-service.test.ts

```diff
@@ -81,6 +96,42 @@ describe('GraphQLDictionaryService', () => {
     expect(result).to.have.all.keys('foo', 'bar');
   });
 
+  it('should use a jssTemplateId, if provided', async () => {
+    const jssAppTemplateId = generateGUID();
+    // mock a random template id
+    const randomId = generateGUID();
```

Comment:
I'm not sure that it makes sense to generate random uid, anyway we are not using real services, basically you can hardcode some id.